draft-normington-jsonpath-latest.md

```diff
@@ -84,43 +108,282 @@ informative:
 
 normative:
   RFC3629:
-  RFC5234:
+  RFC5234: abnf
+  RFC8259: json
 
 --- abstract
 
-
 JSONPath defines a string syntax for identifying values
 within a JavaScript Object Notation (JSON) document.
 
 --- note_
 
 **This document is a work in progress and has not yet been published
-as an Internet Draft.**
+as an Internet Draft** (which needs to be fixed soon).
+
+--- note_Contributing
+
+This document picks up the popular JSONPath specification dated
+2007-02-21 and provides a normative definition for it.
+In its current state, it is a strawman document showing what needs to
+be covered.
+
+Comments and issues can be directed at the github repository *insert
+repo here* as well as (for the time when the more permanent home is
```

Comment:
Was this an unfulfilled todo for this PR, or is it supposed to remain open for now?

Reply:
I would put that in when the document is at its final repo.

Reply:
Thanks @cabo. That's what I expected.